src/smart-components/user/add-user-to-group/add-user-to-group-wizard.js

```diff
@@ -0,0 +1,293 @@
+import React, { useRef, useContext, useState, useEffect, memo } from 'react';
+import { Modal, Button } from '@patternfly/react-core';
+import { addPrincipalsToGroup } from '../../../helpers/group/group-helper';
+import FormRenderer from '@data-driven-forms/react-form-renderer/form-renderer';
+import Pf4FormTemplate from '@data-driven-forms/pf4-component-mapper/form-template';
+import { createRows } from '../../group/group-table-helpers';
+import { applyFiltersToUrl, areFiltersPresentInUrl, syncDefaultFiltersWithUrl } from '../../../helpers/shared/filters';
+import PropTypes from 'prop-types';
+import { WarningModal } from '../../common/warningModal';
+import { addNotification } from '@redhat-cloud-services/frontend-components-notifications/';
+import GroupRowWrapper from '../../group/group-row-wrapper';
+import { sortable } from '@patternfly/react-table';
+import useFormApi from '@data-driven-forms/react-form-renderer/use-form-api';
+import { shallowEqual, useSelector, connect, useDispatch } from 'react-redux';
+import { createQueryParams } from '../../../helpers/shared/helpers';
+import paths from '../../../utilities/pathnames';
+import messages from '../../../Messages';
+import pathnames from '../../../utilities/pathnames';
+import {
+  applyPaginationToUrl,
+  defaultAdminSettings,
+  defaultSettings,
+  isPaginationPresentInUrl,
+  syncDefaultPaginationWithUrl,
+} from '../../../helpers/shared/pagination';
+import { useIntl } from 'react-intl';
+import { TableToolbarViewOld } from '../../../presentational-components/shared/table-toolbar-view-old';
+
+import { useFieldApi } from '@data-driven-forms/react-form-renderer';
+import PermissionsContext from '../../../utilities/permissions-context';
+import { fetchGroups, fetchRoleForUser } from '../../../redux/actions/group-actions';
+import { useHistory } from 'react-router-dom';
+import { getUserGroups } from '../../../helpers/group/group-helper';
+
+const FORM_ID = 'add-user-to-group-form';
+
+const FormTemplate = (props) => <Pf4FormTemplate {...props} showFormControls={false} formWrapperProps={{ id: FORM_ID }} />;
+
+const GroupTable = (conf) => {
```

Comment:
Might be better to move it to a separate file (like `user-list`), we could then reuse it also in the Groups page